include/deal.II/base/exceptions.h

```diff
@@ -1286,11 +1286,23 @@ namespace deal_II_exceptions
    * still call abort(), e.g. when an exception is caught during exception
    * handling.
    *
+   * @see enable_abort_on_exception
    * @see Exceptions
    */
   void
   disable_abort_on_exception();
 
+  /**
+   * Calling this function switches on the use of <tt>std::abort()</tt> when
+   * an exception is created using the Assert() macro, instead of throwing it.
+   * This restores the standard behavior.
```

Comment:
What is the standard behavior?

Reply:
The default behavior is to abort when an exception is thrown.

Before this PR, this behavior could be disabled via `disable_abort_on_exception` but there was no analogous way to reset it (except by explicitly setting the public attribute `internals::allow_abort_on_exception = true;`, which is a bit messy in my opinion).